project_W/model.py

```diff
@@ -193,14 +215,14 @@ def _send_email(
     context = ssl.create_default_context()
 
     try:
-        #default instance for unencrypted and starttls 
-        #ssl encrypts from beginning and requires a different instance
-        smtpInstance = (SMTP_SSL(smtpConfig["domain"], smtpConfig["port"], context=context) 
-            if smtpConfig["secure"] == "ssl" 
-            else SMTP(smtpConfig["domain"], smtpConfig["port"]))
+        # default instance for unencrypted and starttls
+        # ssl encrypts from beginning and requires a different instance
+        smtpInstance = (SMTP_SSL(smtpConfig["domain"], smtpConfig["port"], context=context)
+                        if smtpConfig["secure"] == "ssl"
+                        else SMTP(smtpConfig["domain"], smtpConfig["port"]))
```

Comment:
This is the correct formatting according to autopep8. I'm guessing that since it's a ternary expression, it likes to indent all parts of the expression equally, instead of just doing 4 spaces.